### ISSUE TITLE
The Release 3.2.36 IPA is not available to download, is there a release date?

### ISSUE THREAD

**Issue by marcelojulianoramos** [PopcornTimeTV/PopcornTimeTV#901]
tvOS version:

Hello guys,

When we will have the Release 3.2.36 IPA available for to download, the actual version does not work at all.

**Comment by stepo11** [comment]
I agree with this. I see commits been made but no update on the release. I know there are other forked IPA files others have been sharing but it would be good for the official source to be updated.

**Comment by tim426** [comment]
Was trying to compile one myself, give me faults while building the project, will someone update the ipa please. Thanks

**Comment by mohandshamada** [comment]
yes, I can see that also. I don't understand how to compile it. if any one can help me with how to compile it please send me the tutorials as the description on the repo I can't understand it.

**Comment by NSExceptional** [comment]
@tim426 @mohandshamada You need to first install cocoapods on your computer:

```bash
sudo gem install cocoapods
```

Then inside the project folder, run `pod install`

**Comment by f-bone** [comment]
please give more info.. I installed cocapods and run the command. It downloaded some dependencies etc but where is the .ipa?

**Comment by NSExceptional** [comment]
Why did you thumbs down me?

You have to open the project and build it for Generic Device and the IPA will be in the sidebar where it says _Products_.

**Comment by f-bone** [comment]
it wasnt me. thanks for the additional info.. Im now installing the complete Xcode package to build it.

**Comment by senadaruc** [comment]
Hi, Can anybody please compile and upload a 3.2.36 Apple TV version, please?
